develop/api/index.md

```diff
@@ -0,0 +1,223 @@
+---
+name: Develop with the Docker SDK and API
+description: Using the Docker SDK and API to automate Docker tasks in your language of choice
+keywords: developing, api, sdk
+redirect_from:
+- /engine/api/
+- /engine/reference/api/
+- /engine/reference/api/docker_remote_api/
+- /reference/api/
+- /reference/api/docker_remote_api/
+- /engine/api/sdks/
+---
+
+Docker provides an API and SDKs in multiple languages, so that you can build and
+scale Docker apps and solutions quickly and easily, no matter what languages or
+infrastructures your organization uses. The API and SDKs provide programmatic
+access to core Docker functions (previously Docker Engine), as well as Docker
+Registry and Notary. APIs and SDKs for more components will be added in the
+future.
+
+The Docker API is a RESTful API accessed by an HTTP client such as `wget` or
+`curl`, or the HTTP library which is part of most modern programming languages.
+Where possible, SDKs are provided for Python and Go developers, for even more
+convenience than using the API directly.
+
+## Available APIs and SDKs
+
+The following table links you straight to the API and SDK documentation for each
+Docker component with SDK or API coverage.
+
+| Component       | HTTP API | Python SDK | Go SDK |
+|:----------------|:---------|:-----------|:-------|
+| Moby core       |          |            |        |
+| Docker registry |          |            |        |
+| Notary          |          |            |        |
```

Comment:
empty table, I guess this will be filled in before merge?

Reply:
@tiborvass will help me with this.